packages/pangea-sdk/.sdk-ci.yml

```
@@ -48,17 +92,122 @@ testing:
   script:
     - poetry run python -m unittest tests.unit
 
-integration:
-  extends: .pangea-sdk-base
+audit:
+  extends: .pangea-sdk-test-base
   stage: integration_tests
+  variables:
+    ENV: ${SERVICE_AUDIT_ENV}
   script:
-    - poetry run python -m unittest tests.integration
+    - poetry run python -m unittest tests.integration.test_audit
+  # allow_failure should be true just for GCP
+  allow_failure: true
 
-asyncio-integration:
-  extends: .pangea-sdk-base
-  stage: asyncio_tests
+vault:
+  extends: .pangea-sdk-test-base
+  stage: integration_tests
+  variables:
+    ENV: ${SERVICE_VAULT_ENV}
+  script:
+    - poetry run python -m unittest tests.integration.test_vault
+
+authn:
+  extends: .pangea-sdk-test-base
+  stage: integration_tests
+  variables:
+    ENV: ${SERVICE_AUTHN_ENV}
+  script:
+    - poetry run python -m unittest tests.integration.test_authn
+
+embargo:
+  extends: .pangea-sdk-test-base
+  stage: integration_tests
+  variables:
+    ENV: ${SERVICE_EMBARGO_ENV}
+  script:
+    - poetry run python -m unittest tests.integration.test_embargo
+
+redact:
+  extends: .pangea-sdk-test-base
+  stage: integration_tests
+  variables:
+    ENV: ${SERVICE_REDACT_ENV}
   script:
-    - poetry run python -m unittest tests.integration.asyncio
+    - poetry run python -m unittest tests.integration.test_redact
+
+file-scan:
+  extends: .pangea-sdk-test-base
+  stage: integration_tests
+  variables:
+    ENV: ${SERVICE_FILE_SCAN_ENV}
+  script:
+    - poetry run python -m unittest tests.integration.test_file_scan
+  # allow_failure should be true just for GCP
+  allow_failure: true
```

> allow_failure should be true just for GCP

This might be possible like so:

```suggestion
  rules:
    - if: '$CLOUD == "GCP"'
      allow_failure: true
    - if: '$CLOUD != "GCP"'
      allow_failure: false
```